packages/components/src/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import styles from "./Gallery.css";
+import { GalleryProps } from "./GalleryTypes";
+import { LightBox } from "../LightBox";
+import { FormatFile } from "../FormatFile";
+import { Button } from "../Button";
+
+export function Gallery({ files, size = "default", max }: GalleryProps) {
```

Comment:
Similar to the FormatFile, can we change the `default` size to `base`?

Reply:
Hey @chris-at-jobber, I was just about to address this comment, changing this to `base` results in a lot of cascading changes. I can't seem to see where you suggested the changes in FormatFile, it still looks to use `default`

Reply:
Oh, it was left as a requested change [here](https://github.com/GetJobber/atlantis/pull/889#pullrequestreview-994232683) - I had thought it was addressed. We should try and get that fixed here to avoid introducing breaking changes to Jobber.

Reply:
Ah I see. Okay we can do that rename here definitely, just was confused by the comment.

Reply:
Thanks!